src/variorum/variorum.h

```diff
@@ -226,7 +226,6 @@ int variorum_print_verbose_thermals(void);
 /// - Intel Kaby Lake
 /// - Intel Cascade Lake
 /// - Intel Cooper Lake
-/// - Intel Arctic Sound
```

Comment:
Why are these being removed?

Reply:
That looks like a mistake, let me revert that shortly.

Reply:
Actually, not a mistake, @slabasan . Intel Arctic Sound is a GPU, and doesn't support these APIs.